src/app/containers/StoryPromo/index.jsx

```diff
@@ -182,20 +185,24 @@ const StoryPromoContainer = ({
           {promoSummary}
         </Summary>
       )}
-      {timestamp && !isStoryPromoPodcast && !isRecommendation && !isLive && (
-        <Timestamp
-          altCalendar={altCalendar}
-          locale={datetimeLocale}
-          timestamp={timestamp}
-          dateTimeFormat="YYYY-MM-DD"
-          format="LL"
-          script={script}
-          padding={false}
-          service={service}
-          timezone={timezone}
-          isRelative={isTenHoursAgo(timestamp)}
-        />
-      )}
+      {timestamp &&
```

Comment:
Since we have too many conditions atm, I would suggest creating a separate constant like:
```
const displayTimestamp = timestamp && !isStoryPromoPodcast && !isContentTypeGuide && !isRecommendation && !isLive 
```

```suggestion
      {displayTimestamp && (
```